src/winfspy/plumbing/get_winfsp_dir.py

```diff
@@ -47,4 +47,19 @@ def get_winfsp_bin_dir():
     - the `bin` directory in the base directory provided by `get_winfsp_dir``
     """
     path = os.environ.get("WINFSP_DEBUG_PATH")
-    return path if path else get_winfsp_dir(suffix="bin")
+    if path:
+        path = pathlib.Path(path)
```

Comment:
So now `get_winfsp_bin_dir` and `get_winfsp_dir` return `Path`, are we sure there is no place where it still assumes it returns str ?

Reply:
I'll double check!